MANIFEST.in

```diff
@@ -46,6 +46,7 @@ exclude .coveragerc
 exclude *.yml
 exclude ignore_words.txt
 exclude .mailmap
+exclude mne/viz/_brain/tests/test.ipynb
```

Comment:
you should include it as otherwise you cannot test the installed library.

Reply:
I will include it then.